content/posts/how-to-use-playwright-in-cucumberjs/index.md

```diff
@@ -0,0 +1,157 @@
++++
+date = ""
+author = "nploi"
+description = "How to use Playwright in cucumberjs"
+title = "How to use Playwright in cucumberjs"
+categories = ["DevSecOps", "Testing"]
+tags = ["bdd", "end-to-end", "cucumber", "automation", "test"]
+slug = "how-to-use-playwright-in-cucumberjs"
++++
+
+This tutorial helps you run `web` automation tests by using Playwright in Cucumber.
```

Comment:
This tutorial helps you run `web` automation tests, using Playwright in Cucumber.